src/pages/rayong/Main/Main.js

```diff
@@ -1,7 +1,36 @@
 import React from 'react';
+import { useState, useEffect } from 'react';
```

Comment:
```suggestion
import React, { useState, useEffect } from 'react';
```
같은 라이브러리라면 같은 줄에 사용해도 됩니다!